### PR TITLE
Withdraw a bid

### LINKED ISSUE #1937: Withdraw (Cancel) a bid on an open auction as a bidder
As a bidder I want to withdraw my bid after expiration time passes.

**AC:**

- [ ] Withdraw bid from a past auction - https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=1232%3A76313
- [ ] Withdraw bid from a current auction

### Figma
https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=1247%3A107861

### PR DESCRIPTION
**Pull request by drillprop**
Fix #1937 

https://www.figma.com/file/sMydmZJlzhK91FTIPOd7D2/Video-page?node-id=1232%3A76313

### CODE REVIEW

File: packages/atlas/src/hooks/useNftState.ts
Comment (eldiegod) [inline packages/atlas/src/hooks/useNftState.ts]: Why is the bidlocking time removed here?

**Review by eldiegod — APPROVED** [review]
Looks good codewise, ~~couldn't test the extrinsic itself as I couldn't place a bid on my own due to a possible QN bug I encountered.~~

Added a question and pre-approved to not block the PR

P.D: Nvm, it was just the faucet funds thing being quite confusing, tested it and it works awesome.

P.D2: Except if you try to withdraw immediately, it will error as `bidLockingTime` has not passed like I mentioned in a comment below

**Comment by eldiegod** [comment]
regarding the `bidLockingTime` I'm not sure what's the best solution here, maybe disabling the button until it has passed is better than showing no button.

maybe chipin @toiletgranny ?

**Comment by drillprop** [comment]
> regarding the `bidLockingTime` I'm not sure what's the best solution here, maybe disabling the button until it has passed is better than showing no button.
> 
> maybe chipin @toiletgranny ?

Ok, I'll ask Adam what to do here.

**Comment by drillprop** [comment]
Just FYI, talked to Adam, and I know how to resolve the problem. I'll update this PR once #2372 is merged, though, because we're basically working on the same lines of code.

**Comment by drillprop** [comment]
@eldiegod you basically fixed my problem in your previous PR. We will just show 
image 
instead of a button when someone just withdraws a bid.
Added a couple of improvements though. Let me know what you think.

**Review by eldiegod — APPROVED** [review]

File: packages/atlas/src/hooks/useNftTransactions.ts
Comment (kdembler) [inline packages/atlas/src/hooks/useNftTransactions.ts]: This can now be done via `snackbarSuccessMessage` option to `handleTransaction`

**Review by kdembler — APPROVED** [review]

**Pull request merged**
